src/Fantomas.Core/CodeFormatter.fsi

```diff
@@ -12,15 +12,41 @@ type CodeFormatter =
     /// Format an abstract syntax tree using an optional source for trivia processing
     static member FormatASTAsync: ast: ParsedInput * ?source: string * ?config: FormatConfig -> Async<FormatResult>
 
+    /// <summary>
+    /// Format a source string using an optional config.
+    /// </summary>
+    /// <param name="isSignature">Determines whether the F# parser will process the source as signature file.</param>
+    /// <param name="source">F# source code</param>
+    static member FormatDocumentAsync:
+        isSignature: bool * source: string -> Async<FormatResult>
+
     /// <summary>
     /// Format a source string using an optional config.
     /// </summary>
     /// <param name="isSignature">Determines whether the F# parser will process the source as signature file.</param>
     /// <param name="source">F# source code</param>
     /// <param name="config">Fantomas configuration</param>
+    static member FormatDocumentAsync:
+        isSignature: bool * source: string * config: FormatConfig -> Async<FormatResult>
+
+    /// <summary>
+    /// Format a source string using an optional config.
+    /// </summary>
+    /// <param name="isSignature">Determines whether the F# parser will process the source as signature file.</param>
+    /// <param name="source">F# source code</param>
     /// <param name="cursor">The location of a cursor, zero-based.</param>
     static member FormatDocumentAsync:
-        isSignature: bool * source: string * ?config: FormatConfig * ?cursor: pos -> Async<FormatResult>
+        isSignature: bool * source: string * cursor: pos -> Async<FormatResult>
```

Comment:
This overload seems excessive and I don't see it being used in practice.
If you want to use this then you should use the next one and pass the default config.
Please remove it.